src/Adobe-Glyph.xml

```diff
@@ -0,0 +1,37 @@
+<SPDX name="Adobe Glyph List License" identifier="Adobe-Glyph" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/MIT#AdobeGlyph</url>
+  </urls>
+  <license>
+    <copyright>
+      <p>Copyright (c) 1997,1998,2002,2007 Adobe Systems Incorporated</p>
+    </copyright>
+    <body>
+      <p>Permission is hereby granted, free of charge, to any person obtaining a copy of this documentation file
+         to use, copy, publish, distribute, sublicense, and/or sell copies of the documentation, and to permit
+         others to do the same, provided that:</p>
+      <list>
+        <li>
+          <b>-</b>
+          <p>No modification, editing or other alteration of this document is allowed; and</p>
+        </li>
+        <li>
+          <b>-</b>
+          <p>The above copyright notice and this permission notice shall be included in all copies of the
+             documentation.</p>
+        </li>
+      </list>
+      <p>Permission is hereby granted, free of charge, to any person obtaining a copy of this documentation file,
+         to create their own derivative works from the content of this document to use, copy, publish,
+         distribute, sublicense, and/or sell the derivative works, and to permit others to do the same,
+         provided that the derived work is not represented as being a copy or version of this document.</p>
+      <p>Adobe shall not be liable to any party for any loss of revenue or profit or for indirect, incidental,
+         special, consequential, or other similar damages, whether based on tort (including without limitation
+         negligence or strict liability), contract or other legal or equitable grounds even if Adobe has been
+         advised or had reason to know of the possibility of such damages.Ê The Adobe materials are provided on
```

Comment:
Extra character "Ê" appears here and on the next line as well. This seems to come from the source/template file

Also appears in source: http://spdx.org/licenses/Adobe-Glyph.html and https://fedoraproject.org/wiki/Licensing:MIT?rd=Licensing/MIT#AdobeGlyph

Maybe this is a known issue and we just leave it?